src/tenant/admin.py

```diff
@@ -109,5 +112,49 @@ def get_queryset(self, request):
                     tenant.update_cached_fields()
         return qs
 
+    @admin.action(description="Enable google signin for tenant(s)")
+    def enable_google_signin(self, request, queryset):
+        from siteconfig.models import SiteConfig
+
+        try:
+            SocialApp.objects.get(provider='google')
+        except SocialApp.DoesNotExist:
+            self.message_user(request, "You must create a SocialApp record with 'google' as the provider", messages.ERROR)
+            return
+
+        queryset = queryset.exclude(schema_name=get_public_schema_name())
+        for tenant in queryset:
+            with tenant_context(tenant):
+                config = SiteConfig.get()
+                with transaction.atomic():
+                    config._propagate_google_provider()
+                    config.enable_google_signin = True
+                    config.save()
+
+        enabled_count = queryset.count()
+        self.message_user(request, ngettext(
+            "%d tenant google signin was enabled successfully. Please ensure that the tenant domain is added in the Authorized Redirect URIs",
```

Comment:
`... Ensure that the {tenant_domain} is added in the Authorized Redirect URIs on Bytedeck's Google credentials.`

Can you put the {tenant_domain} here, whatever it is for easy copy/paste?

Reply:
Does this look okay-ish?

<img width="1661" alt="image" src="https://user-images.githubusercontent.com/10972027/230760626-75efcd20-adc8-4505-8f68-f02378227f1b.png">

Reply:
Yes! But please add "..on Bytedeck's Google account credentials." to the end, because the person doing this will not be a developer and the prompt will help, especially if we don't get this request (to turn on Google sign in) very often.